apps/emqx/src/emqx_cm.erl

```diff
@@ -723,6 +723,7 @@ do_get_chann_conn_mod(ClientId, ChanPid) ->
 mark_channel_connected(ChanPid) ->
     ?tp(emqx_cm_connected_client_count_inc, #{chan_pid => ChanPid}),
     ets:insert_new(?CHAN_LIVE_TAB, {ChanPid, true}),
+    ?tp(emqx_cm_connected_client_count_inc_done, #{chan_pid => ChanPid}),
```

Comment:
Maybe should be enough just to move the tracepoint, instead of introducing a new one? Feels a bit too much for such a trivial operation.

Reply:
I think you are right. I saw that `mark_channel_disconnected` had `emqx_cm_connected_client_count_dec` and `emqx_cm_connected_client_count_dec_done` and I thought it made sense to make them symmetrical but there is probably no good reson for that so I will just move it instead of introducing a new one (it might even fix some other flakiness).

Reply:
Fixed and force pushed